packages/web-components/src/appointment-planner/client/components/time-cell.svelte

```diff
@@ -0,0 +1,26 @@
+<script>
+  import { css } from 'emotion'
+  import cx from 'classnames'
+
+  export let startTime = ''
+
+  const backgroundStyle = css`
+    background: ${startTime ? '#e6e6e6' : ''};
+    &:hover {
+      background: ${startTime ? 'var(--hoverBackgroundColor)' : ''};
+    }
+  `
+</script>
+
+<style>
+  .time-cell__container {
```

Comment:
BEM

Reply:
@willmcvay I don't understand this comment. Could you help me to clarify it? Thanks

Reply:
Was just another use of BEM - as per my previous, can we remove as we're not composing classes to increase specificity so it's not adding anything. Hopefully makes sense

Reply:
@phmngocnghia you can refer here http://getbem.com/introduction/